### ISSUE TITLE
[`blocks-dev`] Audio output doesn't seem to play audio

### ISSUE THREAD

**Issue by abidlabs** [gradio-app/gradio#896]
### Describe the bug

I'm running a demo which produces an audio output (`demo\generate_tone\run.py`), but all I see is a grayed audio player (see screenshot).

<img width="981" alt="image" src="https://user-images.githubusercontent.com/1778297/160485848-8665e3d7-15ab-4e45-a70b-aab277dc8aab.png">

I've verified that the Interface returns a valid base64 audio file, so the problem must be in the Audio component itself

### Reproduction

`demo\generate_tone\run.py`

### System Info

```shell
`blocks-dev`
```

### Severity

critical

**Comment by abidlabs** [comment]
Same issue happening in: `demo\reverse_audio\run.py`

**Comment by abidlabs** [comment]
Or try something like:

```python
bl = gr.Blocks()
with bl:
    gr.Audio("cantina.wav")    
    bl.launch()
```

**Comment by abidlabs** [comment]
I think Video component also doesn't play Video fyi:
```python
bl = gr.Blocks()
with bl:
    gr.Video("test.mp4")    
    bl.launch()
```

**Comment by abidlabs** [comment]
Still seeing some issues here (no audio is shown):

<img width="680" alt="image" src="https://user-images.githubusercontent.com/1778297/161814170-b5e2a05d-db7a-47c5-b182-9b2713c03214.png">

Even in an interface, there seems to be an issue with Audio output (instead of an output, an input is shown):

<img width="744" alt="image" src="https://user-images.githubusercontent.com/1778297/161814278-ad798871-9789-48be-9154-5fc1ae78ee06.png">

**Comment by pngwn** [comment]
Closed by #930.
